aten/src/ATen/CMakeLists.txt

```diff
@@ -264,11 +247,12 @@ IF(USE_CUDA AND NOT USE_ROCM)
 ENDIF()
 
 IF(USE_ROCM)
- ### Link in the ROCm libraries BLAS / RNG.
+ ### Link in the ROCm libraries BLAS / RNG / FFT.
  FIND_LIBRARY(ROCBLAS_LIBRARY rocblas HINTS ${ROCBLAS_PATH}/lib)
  FIND_LIBRARY(HIPRAND_LIBRARY hiprand HINTS ${HIPRAND_PATH}/lib)
+ FIND_LIBRARY(ROCFFT_LIBRARY rocfft HINTS ${ROCFFT_PATH}/lib)
```

Comment:
We actually shouldn't put this here in ATen/CMakeLists.txt which is slowly being deprecated. Better to add to pytorch/cmake/public/LoadHIP.cmake & pytorch/cmake/Dependencies.cmake.